recipes/qt/6.x.x/conanfile.py

```diff
@@ -116,6 +118,7 @@ class QtConan(ConanFile):
         "gui": True,
         "widgets": True,
 
+        "android_sdk": None,
```

Comment:
what do you think about setting default value to `ANDROID_SDK_ROOT` env var to be inline with Qt default behavior? https://github.com/qt/qtbase/blob/ced8727a1e5aa0edbe1ab92ba41a7f5b90b783ad/cmake/QtToolchainHelpers.cmake#L272